config/initializers/menus.rb

```diff
@@ -364,7 +368,7 @@
 
   menu.push :backups,
             { controller: '/admin/backups', action: 'show' },
-            if: Proc.new { OpenProject::Configuration.backup_enabled? && User.current.admin? },
+            if: Proc.new { OpenProject::Configuration.backup_enabled? && User.current.allowed_to_globally?(Backup.permission) },
```

Comment:
I forgot to mention that doing `Backup.permission` here during initialization will load `Backup` class and lead to the infamous message `DEPRECATION WARNING: Initialization autoloaded the constants [...], Backup, ...`.

We could do it if the menu initialization was not part of the Rails initializers but we're not there yet.